repo_tests/python_filter_covers_its_guards_test.py

```
@@ -0,0 +1,161 @@
+    the filter grew to cover it reads as a live bypass that is not one, and the
+    next person spends the same afternoon re-deriving why.
+    """
+    uncovered, _ = _uncovered_reads(_filter_patterns())
+    assert len(uncovered) <= MAX_UNCOVERED_READS, (
+        f"{len(uncovered)} uncovered guard inputs against a ceiling of {MAX_UNCOVERED_READS}"
+    )
+
+    stale = sorted(path for path in UNCOVERED_READS if path not in uncovered)
+    assert not stale, (
+        "the filter now covers these, so they are no longer bypasses — remove them from "
+        f"UNCOVERED_READS and lower MAX_UNCOVERED_READS to match:\n  " + "\n  ".join(stale)
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)